main/guides/getting-started/Explainer - Deploying a Smart Contact.md

```diff
@@ -0,0 +1,22 @@
+# Deploying a Smart Contact
```

Comment:
This is redundant with the /getting-started/ page. Please let's not duplicate documentation this way.

Reply:
This is what @sufyaankhan wanted documented...